requirements/edx/github.txt

```diff
@@ -92,7 +92,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 -e git+https://github.com/edx/edx-reverification-block.git@0.0.5#egg=edx-reverification-block==0.0.5
 -e git+https://github.com/edx/edx-user-state-client.git@30c0ad4b9f57f8d48d6943eb585ec8a9205f4469#egg=edx-user-state-client
-git+https://github.com/edx/edx-organizations.git@release-2016-01-22#egg=edx-organizations==0.3.0
+git+https://github.com/edx/edx-organizations.git@release-2016-01-26#egg=edx-organizations==0.3.0
```

Comment:
This package is on PyPI. All you need is `edx-organizations==0.3.0` in `requirements/edx/base.txt`.